### PR TITLE
Slight clarification on getting started guide

### PR DESCRIPTION
**Pull request by mr-vaibh**
Updating readme so that newcomers don't find the setup guide confusing.

[^1]: [Development Testing](https://docs.wagtail.org/en/latest/contributing/developing.html#testing)
[^2]: [Browser and device support](https://docs.wagtail.org/en/latest/contributing/developing.html#browser-and-device-support)
[^3]: [Accessibility Target](https://docs.wagtail.org/en/latest/contributing/developing.html#accessibility-targets)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,7 +41,8 @@ python manage.py createsuperuser
 python manage.py runserver
 ```
 
-For detailed installation and setup docs, see [docs.wagtail.org](https://docs.wagtail.org/).
+Note: Above commands are the implementation of Wagtail, NOT setup guide.
```
Comment (gasman) [inline README.md:44]: I'm not sure that this really makes things clearer... I don't know what "the implementation of Wagtail" is meant to mean (to me that would mean the Python code of Wagtail itself), and a beginner would be even more confused about why this is here, if it's something else besides setup instructions.

Maybe this is just missing an instruction to go to `http://localhost:8000`, which will then bring up the welcome page with further documentation links. Or maybe we don't really need this section at all, and should just link to the Getting Started page of the docs. I understand there's some marketing appeal in saying "Wagtail can be installed in just 7 lines", but that's already on the website, and the Getting Started docs do a better job of guiding users through setup, including setting up a virtual environment.

cc @KalobTaulien who's working on some wider improvements to the readme.

**Review by gasman — CHANGES_REQUESTED** [review]

**Comment by mr-vaibh** [comment]
@gasman Hey, I got what you were saying. What I meant was....there are two types of setup a user might want to do

The first one is to USE the Wagtail to create their website, (where the person is a user)
The second one is to set up the CODEBASE on their PC in order to contribute. (where the person is contributer)

I myself found it very difficult to set up the source code.
So, keeping that in mind I've proposed these changes.

Please give me further feedback if necessary.

**Comment by gasman** [comment]
Thanks, I understand now!

During initiatives like GSoC and Hacktoberfest, people might be arriving at the project with the intention to contribute, but that's a relatively rare case - the vast majority just want to use it (and will _maybe_ contribute later on if there's a specific thing they want to fix), and for those people, even mentioning the option of contributing is a distraction at this point in the readme. There's a "Contributing" section further down the readme addressing the people who do want to contribute.

I'd be happy to go for your first change ("To get started with Wagtail" -> "To get started with using Wagtail") - that one seems like a good idea, since the meaning stays exactly the same for 'normal' users, while giving GSoC / Hacktoberfest people extra information to help steer them to the right place 👍 . I think the rest should be kept as the original version.

**Comment by mr-vaibh** [comment]
@gasman Good good. I've reverted the 2nd part of the changes. It should be fine now.

**Pull request merged**